include/lsst/afw/geom/Angle.h

```diff
@@ -246,73 +299,182 @@ Angle const NullAngle = Angle(-1000000., degrees);
  * N.b. We need both int and double versions to avoid ambiguous overloading due to implicit conversion of
  * Angle to double
```

Comment:
`ANGLE_OP` does not appear to do anything with `int`; I suspect this comment refers to `ANGLE_OP_TYPE`; if so please move it.